public/src/js/components/StripeCheckoutComponent.js

```diff
@@ -21,20 +21,20 @@ module.exports = React.createBackboneClass({
 
   render: function() {
 
-    var disabled = isNaN(this.getModel().get('paymentAmount')) || this.getModel().get('paymentAmount')<= 0; 
+    var disabled = isNaN(this.getModel().get('paymentAmount')) || this.getModel().get('paymentAmount')<= 0;
 
     return (
       <StripeCheckout
         token={this.onToken}
-        stripeKey={process.env.STRIPE_PUBLISHABLE_KEY}
+        stripeKey="pk_test_LIWkhhJVRCPoGBO3SXaUXmTS"
```

Comment:
We have to be very careful here, as this is a potential monumental mistake. While exposing this particular value isn't the end of the world, the keys in your `.env` file are extremely secret, and can never be committed and introduced into the codebase, especially in a public repository like this. There are places on the deployment and testing server that we can enter them securely.

Now I know this was the default key that stripe gave us to test with, but this would have also broken our production build because it wouldn't have gotten the real keys from the server there. Let me know if you have any more questions about this.